fluentlenium-core/src/main/java/org/fluentlenium/core/inject/ContainerFluentControl.java

```diff
@@ -46,6 +48,16 @@ public ContainerFluentControl(FluentControl adapterControl) {
         this.adapterControl = adapterControl;
     }
 
+    /**
+     * Make control over chromium api.
+     *
+     * @return a new chromium api instance
+     * @param driver remote webdriver instance
+     */
+    public final ChromiumApi makeChromiumApi(RemoteWebDriver driver) {
```

Comment:
It should be in FluentControl interface and in FluentControlImpl instead

Reply:
Ok, now I get it.

Reply:
Fixed.

Reply:
Ok, now I get it.